Exiled.Loader/Loader.cs

```diff
@@ -330,7 +350,8 @@ public static void DisablePlugins()
                 }
                 catch (Exception exception)
                 {
-                    Log.Error($"Plugin \"{plugin.Name}\" threw an exception while disabling: {exception}");
+                    Log.Error(
```

Comment:
Do we need a new line here? (And in every `Log.Error(...)` above)

Reply:
Not really, just Rider's code cleanup, rolling back that mess.